nuxt/layouts/default.vue

```diff
@@ -1,13 +1,25 @@
 <template>
   <div>
-    <nuxt />
+    <the-navbar />
```

Comment:
Pour info :  PascalCase pour les autoclosing tags

Reply:
Yes, merci, j'ai lu https://vuejs.org/v2/style-guide/#Self-closing-components-strongly-recommended et je m'interrogeais sur la phrase : "the strategy is only possible when Vue’s template compiler can reach the template before the DOM, then serve the DOM spec-compliant HTML." Est-ce tout le temps le cas dans une app Nuxt? Ou bien doit-on éviter les self-closing tags? @KevinBerthier